### PR TITLE
Service to set light brightness

### PR DESCRIPTION
**Pull request by tijsverkoyen**
**tl;dr**: Expose a service to set the light brightness without turning them on.

At the moment I have a schedule that defines nighttime. When nighttime is active I want some lights to have a specific brightness. Per example: when I wake up at 03:00 to pee (yeah, I'm old) I want the light at 30% instead of blinding me.

In Home Assistant I have used an Automation that uses the trigger "Off → On", and that uses a [service "Light: Turn on"](https://www.home-assistant.io/integrations/light/#service-lightturn_on) that is able to set the brightness.

But the problem is that the light starts with the previous brightness setting, so if the light was 85% before: it will turn on at 85% and dim to the given brightness (eg: 30%). 

You could think that I had to disable "Dimmer memory", but when you do this the default value is 100%. There is probably a way to configure this with conditions and routines in the Nike Home Control programming software but I thought it would be more logical to have this available in Home Assistant.

### CODE REVIEW

**Comment by tijsverkoyen** [comment]
I have tested it locally over the weekend and it works like intended.

**Comment by tijsverkoyen** [comment]
I also wrote a blogpost about it: https://verkoyen.eu/2022/10/dim-lights-without-turning-them-on/
